commands/grid.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
 	description: "Displays the building grid and all the buildings you have currently built in your empire.",
 	async execute(message, args, user) {
 		const gridImage = await createGridCanvas(user);
-		message.channel.send(`<@${message.author.id}>'s Empire:`, gridImage);
+		await user.save();
```

Comment:
Hvorfor er user.save lagt til her egentlig?

Reply:
Det er for å lagre statistikken. Nå så ser vi ikke om folk bruker !empire eller !grid fordi det blir aldri lagret.

Reply:
Det blir forsåvidt lagt til i index.js, men det er ikke noe user.save der i tilfelle det går inn i multiple save document issue

Reply:
Aight!